test/analysis_ee_ZH_cfg.py

```diff
@@ -12,6 +12,7 @@
 import copy
 import heppy.framework.config as cfg
 import heppy.utils.pdebug
+from heppy.papas.data.identifier import Identifier
```

Comment:
This is not needed here, only in the main section (but even, such technical details should not be left to the responsibility of the user)